docs/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
```

Comment:
Docker compose files should now follow [Docker Compose specification](https://docs.docker.com/compose/compose-file/), without version.

Reply:
Oh didn't known about that, fixed everywhere plus add a `Note` about it